Google Cloud/google-drive-reports/ingest/parser.yml

```diff
@@ -10,11 +10,13 @@ stages:
   set_ecs_fields:
     actions:
       - set:
+          "@timestamp": "{{json_event.message.id.time}}"
           event.kind: "event"
           event.category: ["file"]
-          event.type: ["access"]
+          event.type: ["access", "edit"]
```

Comment:
This should change depending on the event. By default, just `"access" `seems good. 
If we believe [this documentation](https://support.google.com/a/answer/4579696?hl=en) Event name can be set to `View, Rename, Create, Edit, Preview, Print, Update, Delete, Upload, Download, Share a Drive file, or Connected Sheets refreshed objects`